runtime/datetime.cpp

```diff
@@ -372,18 +364,24 @@ string f$date(const string &format, int64_t timestamp) {
 }
 
 bool f$date_default_timezone_set(const string &s) {
-  if (s != string("Etc/GMT-3", 9) && s != string("Europe/Moscow", 13)) {//TODO
-    if (s == string("Etc/GMT-4", 9)) {
-      php_warning("Timezone Etc/GMT-4 is not supported, use Etc/GMT-3 instead");
-      return true;
-    }
-    php_critical_error ("unsupported default timezone \"%s\"", s.c_str());
+  if (strcmp(s.c_str(), "Etc/GMT-3") == 0) {
+    set_default_timezone_id("Etc/GMT-3");
+    return true;
+  }
+  if (strcmp(s.c_str(), "Europe/Moscow") == 0) {
```

Comment:
nit: Copy-paste of the magic strings makes me cry.
"Europe/Moscow" - 5 times
"Etc/GMT-3" - 5 times
"Etc/GMT-4" - 2 times

May be it's worth to move them into const char* global constants into one place? 
E.g. declare in timelib_wrapper.h, define and use in timelib_wrapper.cpp, and use in datetime.cpp.

Reply:
Done.